integration-tests/__tests__/stack-trace-source-maps.test.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+'use strict';
+
+const {readFileSync} = require('fs');
+const path = require('path');
+const skipOnWindows = require('../../scripts/SkipOnWindows');
```

Comment:
Please rename `skipOnWindows` -> `SkipOnWindows`
See #5511 for more information

Reply:
Will do

Reply:
Done!